src/litdata/processing/functions.py

```diff
@@ -100,7 +100,7 @@ def _get_default_num_workers() -> int:
     return os.cpu_count() or 1
 
 
-class LambdaDataTransformRecipe(DataTransformRecipe):
+class LambdaMapRecipe(MapRecipe):
```

Comment:
same question about the alias for backwards compatibility

Reply:
This is internal.

Reply:
I definitely like MapRecipe better